elements/pl-dataframe/pl-dataframe.py

```diff
@@ -0,0 +1,104 @@
+from typing import cast
+
+import chevron
+import lxml.html
+import pandas
```

Comment:
Any reason not to use the ubiquitous `pd` alias?

Reply:
Used the full name because that's what was done in the python file this was originally based on. Will use the alias since that seems to be more generally accepted.